src/calendar/BookingLocation.tsx

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable react/jsx-no-target-blank */
 import { Button } from "baseui/button";
+import isToday from "date-fns/isToday";
+import isAfter from "date-fns/isAfter";
```

Comment:
Can these two be imported in one line like `import { isToday, isAfter } from 'date-fns`?

Reply:
Changes done